Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -51,6 +51,11 @@
 	fprintf('%s\n', 'Performing multi-PLD quantification');
 else
 	fprintf('%s\n', 'Performing single-PLD quantification');
+	if nargout > 2
+		warning('Cannot assign more than two output parameters for single-PLD quantification. Setting ATT and Tex to empty vectors.')
```

Comment:
No this is wrong. Single-PLD can output `Tex`. Like below, I would treat single- and multi-PLD the same throughout the code, only at the end you do a check between them

Reply:
Yes true. Single-PLD, multi-TE. I didn't realize that. Will do :)